src/App.js

```diff
@@ -15,11 +15,14 @@ function App() {
       console.log("input: ", input);
     } else if (label === "C" ) {
         setInput((prevInput) => prevInput.slice(0, -1));
+    } else if (['+', '-', '*', '/'].includes(label)) {
```

Comment:
die Operatoren auch sonst gerne auslagern in eine globale Konstante (über `function App`)

const MathOperator = ['+', '-', '*', '/'];

und dann hier 
```suggestion
    } else if (MathOperator.includes(label)) {
```

Reply:
habe es in [aeccb70](https://github.com/jason-roesgen/calculator-react/pull/9/commits/aeccb7069f274582203f7f46e4f4f7497f077d8e) angepasst